extensions/wikia/AdEngine/js/provider/monetizationService.js

```diff
@@ -1,12 +1,15 @@
 define('ext.wikia.adEngine.provider.monetizationService', [
 	'ext.wikia.adEngine.adContext',
-	'wikia.loader',
+	'ext.wikia.adEngine.monetizationsServiceHelper',
+	'jquery',
 	'wikia.log',
-	'wikia.scriptwriter',
-], function (adContext, loader, log, scriptWriter) {
+	'wikia.nirvana',
+	'wikia.window',
+], function (adContext, monetizationService, $, log, nirvana, window) {
```

Comment:
What do you think about renaming `monetizationService` to `monetizationServiceHelper`?